frontend/src/components/taskSelection/taskList.js

```diff
@@ -301,7 +297,7 @@ export function TaskList({
           <PaginatedList
             pageSize={6}
             items={
-              sortBy === 'id' ? readyTasks.sort(compareTaskId) : readyTasks.sort(compareLastUpdate)
+              sortBy === 'id' ? readyTasks.sort(compareTaskId) : sortBy === 'date' ? readyTasks.sort(compareLastUpdate) : (readyTasks.sort(compareLastUpdate)).reverse()
```

Comment:
the line is too long

Reply:
@willemarcel  can  you suggest something else

Reply:
you can just break the lines after the `?` and `:`